Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -614,6 +689,14 @@ void ConnectionFromClient::start_request(i32 request_id, ByteString method, URL:
                 request->curl_string_lists.append(curl_headers);
             }
 
+            if (resume_request.has_value()) {
+                auto range = ByteString::formatted("{}-", resume_request->start_offset);
+                set_option(CURLOPT_RANGE, range.characters());
+
+                request->got_all_headers = true; // Don't re-send the headers for resumed requests.
```

Comment:
Hm, I guess this means we don't update the cache entry's headers based on the (potentially) partial response? But is that the correct behavior, i.e. what should we do if other headers actually change between the initial response and the partial response? An alternative would be to take the headers into account, but leave out `Content-Length` and `Content-Range` if present.